monitoring/types.go

```diff
@@ -0,0 +1,22 @@
+package monitoring
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	longhornName = "longhorn"
+
+	subsystemVolume      = "volume"
+	subsystemBackupStats = "backup_stats"
+	subsystemNodeStats   = "node_stats"
```

Comment:
remove the `stats` suffix and just use `node`
this way it will look nicer and be easier to read:

`node_status` vs `node_stats_node_status`

this also makes sense since all our per component collectors are individual metrics and not aggregates.
If we want to add different aggregated statistics we could add a collector like this instead:
`stats_nodes`
`stats_volumes`

But at the moment I don't see any direct need for this, since once all individual metrics are tracked via prometheus the user can use the prometheus query language to generate the aggregates themselves.
Which is much better then us generating them.

Reply:
Thank you! Fixed